### DISCUSSION TITLE
How to Display data on the timeline?

### DISCUSSION THREAD

**Discussion by zhouxiangpeng** [nhn/tui.calendar#1293] [Q&A] [answered]
(https://stackblitz.com/edit/react-xktdsp?file=src/App.js)[https://stackblitz.com/edit/react-xktdsp?file=src/App.js]
## Version

v2.1.3
## Test Environment

chrome, macos
## Current Behavior

```js
export default function App() {
  const events = [
    {
      id: 'test_event',
      calendarId: 'DONE',
      title: 'TEST EVENT',
      start: '2022-09-27T15:00:00',
      end: '2022-09-27T23:00:00',
      isVisible: true,
      isAllday: true,
      category: 'time',
    },
  ];
  return (
    <div>
      <Calendar
        height="100vh"
        view={'week'}
        useDetailPopup={true}
        events={events}
      />
    </div>
  );
}
```

## Expected Behavior

I don’t know how to display the data on the timeline, you can help me see where this is wrong？ thanks

**Comment by adhrinae** [comment, marked as answer]
You set the `isAllday` property to `true`.

https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/event-object.md#isallday

**Reply by zhouxiangpeng** [reply]
thanks
